be/src/common/config.h

```diff
@@ -35,8 +35,10 @@ CONF_Int32(be_port, "9060");
 // port for brpc
 CONF_Int32(brpc_port, "8060");
 
-// the number of bthreads for brpc, the default value is set to -1, which means the number of bthreads is #cpu-cores
-CONF_Int32(brpc_num_threads, "-1");
+// the number of bthreads for brpc, the default value is set to 32
+// brpc only for network service send or accept request
+// no more  process any logic
+CONF_Int32(brpc_num_threads, "32");
```

Comment:
We do not need to change default value here, e.g. if a user runs server on a machine with 128 cores, it would break his/her config used.

Reply:
brpc now only for network  i/o, i think no need to configure so many threads

Reply:
brpc creates a pthread for each cpu, this is a default behavior, and we do not care it on any machine if we use it in a right way. I can not see any benefits on changing it.